### DISCUSSION TITLE
NLP

### DISCUSSION THREAD

**Discussion by Dexterr-net** [community/community#123078] [Programming Help]
i want to make an nlp regarding analyzing resume

**Comment by Dhruv-net** [comment]
Please go through my repos you will definately find something for sure

**Comment by goldieowner17** [comment]
Hey there! 👋

Thanks for posting in the GitHub Community, @Dexterr-net! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Discover category is a place for GitHubbers where we post blogs, articles, best practices, and tips & tricks from GitHub employees and users. 

I've gone ahead and moved this to the correct category for you. Good luck!
